include/mysql/psi/mysql_thread.h

```diff
@@ -1320,15 +1320,15 @@ static inline void inline_mysql_thread_set_psi_THD(THD *thd)
 }
 #endif /* __cplusplus */
 
+#endif
```

Comment:
can you comment that is closer of which if

Reply:
Why we are moving up the enclose ifdef HAVE_PSI_THREAD_INTERFACE ?

PXB should have it as false and do not compile this part of code.